pwa-devdocs/scripts/create-reference-docs/config/peregrine/index.js

```diff
@@ -0,0 +1,16 @@
+module.exports = [
+    {
+        target: 'peregrine/src/Price/Price.js'
+    },
+    {
+        target: 'peregrine/src/List/list.js',
+        overrides: {
```

Comment:
What does `overrides` mean / do ?

Reply:
It's meant to override the properties returned by `react-docgen`.

I found that on occasion, a prop that is marked as required in the code does not get marked as such by the parser.